frontend/src/Utils.js

```diff
@@ -17,24 +17,62 @@ export const fetchSongData = async (songName, setCurrSong) => {
   }
 };
 
-export const fetchSonsgByName = async (songName, setSongs, count = 6) => {
-  try {
-    const response = await axios.get(`https://spring-music-player-3hyj.vercel.app/search?song=${encodeURIComponent(songName)}`);
-    const jsonData = response.data;
-    const songs = jsonData.slice(0, count);
-    setSongs(songs);
-  } catch (error) {
-    console.error("There was a problem with the fetch operation:", error);
-  }
+// fetch song by its name
+export const fetchSongsByName = async (songName, setSongs, count = 6) => {
+  fetch(
+    `https://jio-savaan-private.vercel.app/api/search/songs?query=${encodeURIComponent(songName)}`
+  )
+    .then((response) => {
+      if (!response.ok) {
+        throw new Error("Network response was not ok");
+      } else {
+        return response.json();
+      }
+    })
+    .then((response) => {
+      const songs = [];
+      for (let i = 0; i < count; i++) {
+        songs.push(response.data.results[i]);
+      }
+      setSongs(songs);
+    })
+    .catch((error) => {
+      console.error("There was a problem with the fetch operation:", error);
+    });
 };
 
-export const fetchTopSongs = async (setTopSongs, count = 6) => {
+// fetch top songs
+export const fetchTopSongs = async (setTopSongs, count = 20) => {
   try {
-    const response = await axios.get("https://spring-music-player-3hyj.vercel.app/search?song=top songs");
-    const jsonData = response.data;
-    const topSongs = jsonData.slice(0, count);
-    setTopSongs(topSongs);
+    const response = await axios.get("https://jio-savaan-private.vercel.app/api/search/songs?query=top songs");
```

Comment:
dont make changes in util.js

Reply:
> dont make changes in util.js
okay..
this I made changes..otherwise I was not able to fetch properly songs for musicPlayer